### ISSUE TITLE
Apply rustfmt to the codebase

### ISSUE THREAD

**Issue by Delapouite** [Shizcow/dmenu-rs#48]
Hello.

While I was skimming through the codebase through github.com, I was surprised to stumble upon very weirdly indented code which was very hard to follow : 

Here's an example take from `item.rs` : 

![image](https://user-images.githubusercontent.com/39315/213402469-93b60958-dd2d-45a7-9cee-9d4e56cc7b03.png)

Indeed, after git cloning the repo on my local station, it turns out that the source code uses a mixed bag for spaces and tabs, so the problem remain in my text-editor.

I reckon it could be worthwhile to perform some cleanup with `rustfmt` across all rust files in order to alleviate the problem.

Thanks.

**Comment by Shizcow** [comment]
I've added `make fmt` which does the pre-requisite setup and then calls rustfmt. Unfortunately due to how the build system is set up, adding a github action to do this is not easily possible.
